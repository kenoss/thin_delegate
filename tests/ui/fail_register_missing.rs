#[thin_delegate::derive_delegate(ToString)]
enum Hoge {
    A(String),
    B(char),
}

fn main() {}
