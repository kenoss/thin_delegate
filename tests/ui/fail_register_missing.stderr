error: trait not registered: path = ToString

       hint: Add `#[thin_delegate::register(ToString)]` for trait `ToString`
 --> tests/ui/fail_register_missing.rs:1:1
  |
1 | #[thin_delegate::derive_delegate(ToString)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `thin_delegate::derive_delegate` (in Nightly builds, run with -Z macro-backtrace for more info)
